Sources/Danger/GitHubDSL.swift

```diff
@@ -17,489 +13,449 @@ public struct GitHub: Decodable {
+extension GitHub {
+    public struct Milestone: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
+            case closedAt = "closed_at"
+            case closedIssues = "closed_issues"
+            case createdAt = "created_at"
+            case creator
+            case description
+            case dueOn = "due_on"
+            case id
+            case number
+            case openIssues = "open_issues"
+            case state
+            case title
+            case updatedAt = "updated_at"
+        }
 
-    public enum MilestoneState: String, Decodable {
-        case open
-        case closed
-        case all
-    }
+        public enum State: String, Decodable {
+            case open
+            case closed
+            case all
+        }
 
-    // MARK: - Properties
+        /// The id number of this milestone
+        public let id: Int
 
-    /// The id number of this milestone
-    public let id: Int
+        /// The number of this milestone
+        public let number: Int
 
-    /// The number of this milestone
-    public let number: Int
+        /// The state of this milestone: open, closed, all
+        public let state: State
 
-    /// The state of this milestone: open, closed, all
-    public let state: MilestoneState
+        /// The title of this milestone
+        public let title: String
 
-    /// The title of this milestone
-    public let title: String
+        /// The description of this milestone.
+        public let description: String?
 
-    /// The description of this milestone.
-    public let description: String?
+        /// The user who created this milestone.
+        public let creator: User
 
-    /// The user who created this milestone.
-    public let creator: GitHubUser
+        /// The number of open issues in this milestone
+        public let openIssues: Int
 
-    /// The number of open issues in this milestone
-    public let openIssues: Int
+        /// The number of closed issues in this milestone
+        public let closedIssues: Int
 
-    /// The number of closed issues in this milestone
-    public let closedIssues: Int
+        /// The ISO8601 date string for when this milestone was created.
+        public let createdAt: Date
 
-    /// The ISO8601 date string for when this milestone was created.
-    public let createdAt: Date
+        /// The ISO8601 date string for when this milestone was updated.
+        public let updatedAt: Date
 
-    /// The ISO8601 date string for when this milestone was updated.
-    public let updatedAt: Date
+        /// The ISO8601 date string for when this milestone was closed.
+        public let closedAt: Date?
 
-    /// The ISO8601 date string for when this milestone was closed.
-    public let closedAt: Date?
-
-    /// The ISO8601 date string for the due of this milestone.
-    public let dueOn: Date?
+        /// The ISO8601 date string for the due of this milestone.
+        public let dueOn: Date?
+    }
 }
```

Comment:
<!--
  0 failure: 
  1 warning:  File should conta...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 461;
-->

- :warning: File should contain 400 lines or less: currently contains 461 (`file_length`)